### ISSUE TITLE
ReLU6 issue

### ISSUE THREAD

**Issue by HKLee2040** [quic/aimet#525]
Some Mobilenet v1 utilize ReLU6 as the activation function. If I replaced ReLU6 with ReLU, the accuracy drop is obvious even I didn't do any compensation and quantization. Do you have any comment on such situation?

**Comment by quic-akhobare** [comment]
Hi @HKLee2040.. 

Yeah this is unfortunate situation for this model. And we have seen this in some instances. ReLU6 to ReLU conversion is specifically needed for CLE (cross-layer equalization) technique. So we have perhaps two options

1. Not use the CLE technique in this particular case. We can check if the quantized accuracy on this model is good. If needed, we could use quantization-aware training.
2. Replace ReLU6 with ReLU in the model. And perform some fine-tuning. It's likely that the accuracy recovers within a few epochs of training.
